Code/Framework/AzFramework/CMakeLists.txt

```diff
@@ -32,7 +34,7 @@ ly_add_target(
             3rdParty::lz4
 )
 
-set(LY_SEARCH_MODE_DEFINE $<$<NOT:$<STREQUAL:"${LY_ARCHIVE_FILE_SEARCH_MODE}","">>:LY_ARCHIVE_FILE_SEARCH_MODE_DEFAULT=${LY_ARCHIVE_FILE_SEARCH_MODE}>)
+set(LY_SEARCH_MODE_DEFINE $<$<NOT:$<STREQUAL:"${LY_ARCHIVE_FILE_SEARCH_MODE}","">>:LY_ARCHIVE_FILE_SEARCH_MODE=${LY_ARCHIVE_FILE_SEARCH_MODE}>)
```

Comment:
this can be simplified with
```
set(LY_SEARCH_MODE_DEFINE $<$<BOOL:"${LY_ARCHIVE_FILE_SEARCH_MODE}">:LY_ARCHIVE_FILE_SEARCH_MODE=${LY_ARCHIVE_FILE_SEARCH_MODE}>)
```

Reply:
Again I repeat using `$<BOOL:0>` results in the value of  evaluating to false.

This would mean if the user actually wanted to use PakOnly mode in release, it wouldn't work as the define wouldn't be "defined"

Reply:
Nevermind, I am wrong.
Using `$<BOOL:"0">` does evaluate to true.
I'll switch to that.